Aliases/coffeescript

```diff
@@ -0,0 +1 @@
+../Formula/coffee-script.rb
```

Comment:
I'm tempted to agree with you @bfontaine; the reason I went with this naming scheme originally is that, while the `git` repo calls it [`coffeescript`](https://github.com/jashkenas/coffeescript), the main entry point for users (`npm`) calls it [`coffee-script`](https://www.npmjs.com/package/coffee-script). I'm fine with either name, so let me know your thoughts and I can make any changes if necessary 😄